src/_deprecated/Checkbox/index.js

```diff
@@ -4,6 +4,9 @@ import * as React from "react";
 
 import Typography from "../Typography";
 import Icon from "./Icon";
+import deprecationWarning from "../../../config/deprecationWarning";
+
+deprecationWarning("Checkbox is deprecated and will be removed in next major release");
```

Comment:
deprecations warnings need to be in `render` function, this results in console warnings no matter what specifically is imported, since it's executed when file is imported, which is exactly what is happening in index:

```
import { Typography } from '...';
```

Reply:
Yes, but if you add it to render then it warns when every render

Reply:
yes, IMHO this is more desired behaviour, early adoptions should be rewarded

No usage of deprecated components => no warning
Many usages of deprecated components => many warnings

Reply:
Imagine `InputText` component, it rerenders everytime when you type into => sooo many warings, but one usage

also in index there isn't importing, it warns due to `*.stories.js` files

Reply:
> also in index there isn't importing, it warns due to *.stories.js files 

https://github.com/kiwicom/orbit-components/pull/174/files#diff-1fdf421c05c1140f6d71444ea2b27638R7

Reply:
It would be batched together in such case

Reply:
OK, refactored, according to this i had to create new file inside components folder that includes pure component with export without HOC due to tests, because i couldn't get into component via `shallow` ¯\_(ツ)_/¯

Also left `index.js` file with HOC export, so it shouldn't break anything